extensions/mypy_extensions.py

```diff
@@ -8,15 +8,65 @@
 # NOTE: This module must support Python 2.7 in addition to Python 3.x
 
 
-def TypedDict(typename, fields):
-    """TypedDict creates a dictionary type that expects all of its
+import sys
+from typing import _type_check  # type: ignore
```

Comment:
I am using a non-public API here, but probably it is better than just copy the whole function here.